### PR TITLE
feat: integrate stellar wallet

### PR DESCRIPTION
**Pull request by justinenerio**
## Changes

integrates stellar wallet

## Checklist

- [x] PR is ready for review (if not, it should be a draft).
- [x] PR title follows [Conventional Commits][1] guidelines.
- [ ] Screenshots/video added.
- [ ] Tests added.
- [x] Self-review done.

[1]: https://www.conventionalcommits.org/en/v1.0.0/

### CODE REVIEW

File: packages/espressocash_app/pubspec.yaml [hunk 1]
```diff
@@ -84,7 +85,7 @@ dev_dependencies:
   flutter_driver:
     sdk: flutter
   flutter_gen_runner: ^5.4.0
-  flutter_native_splash: ^2.3.8
+  flutter_native_splash: ^2.3.7
```
Comment (ookami-kb) [inline packages/espressocash_app/pubspec.yaml:88]: Why does it need to be downgraded?

File: packages/espressocash_app/ios/Podfile.lock [hunk 2]
```diff
@@ -336,4 +336,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: c2878e103071f1a812f2f4e4c83a762aef964055
 
-COCOAPODS: 1.15.2
+COCOAPODS: 1.13.0
```
Comment (ookami-kb) [inline packages/espressocash_app/ios/Podfile.lock:339]: Shouldn't be downgraded.

Comment (justinenerio) [reply packages/espressocash_app/ios/Podfile.lock]: reverted

File: packages/espressocash_app/pubspec.yaml (hunk 1 above)
Comment (justinenerio) [reply packages/espressocash_app/pubspec.yaml:88]: it has dependency conflicts when adding `stellar_flutter_sdk`. the cli recommends downgrading to that version to resolve

**Review by ookami-kb — APPROVED** [review]

**Pull request merged**